en: Update faq.md (#514)
pingcap/docs-dm: en/faq.md

Currently, DM only supports decoding the standard version of MySQL or MariaDB binlog. It has not been tested for Alibaba Cloud RDS or other cloud databases. If you are confirmed that its binlog is in standard format, then it is supported.

> It is known that even if an upstream table in in Alibaba Cloud RDS has no primary key, its binlog still contains a hidden primary key column, which is inconsistent with the original table structure.

Reviewer: ```suggestion
It is a known issue that for an upstream table in Alibaba Cloud RDS which has no primary key, its binlog still contains a hidden primary key column, which is inconsistent with the original table structure.
```

不是很懂，但是感觉 even if 有点太强烈了 😂

Author: ```suggestion
It is a known issue that for an upstream table with no primary key in Alibaba Cloud RDS, its binlog still contains a hidden primary key column, which is inconsistent with the original table structure.
```
Good advice 👍🏻 But the edited version has two "which" clauses in one sentence. How about this one?

Reviewer: sgtm

## Why does the `replicate lag` monitor metric show no data when DM is not replicating from upstream?

In DM 1.0, you need to enable `enable-heartbeat` to generate the monitor data. In DM 2.0, it is expected to have no data in the monitor metric `replicate lag` because this feature is not supported.

> ## How to handle the error`fail to initial unit Sync of subtask` when DM is starting a task, with the `RawCause` in the error message showing `context deadline exceeded`?

Reviewer: ```suggestion
## How to handle the error `fail to initial unit Sync of subtask` when DM is starting a task, with the `RawCause` in the error message showing `context deadline exceeded`?
```

This is a known issue in DM 2.0.0 version and will be fixed in DM 2.0.1 version. It is likely to be triggered when a replication task has a lot of tables to process. If you use TiUP to deploy DM, you can upgrade DM to the nightly version to fix this issue. Or you can download the 2.0.0-hotfix version from [the release page of DM](https://github.com/pingcap/dm/releases) on GitHub and manually replace the executable files.

## How to handle the error `duplicate entry` when DM is replicating data?

You need to first check and confirm the following things:

- `disable-detect` is not configured in the replication task.

> - The data is not inserted manually or by other replication procedures.

Reviewer: "procedures" 好像不太常用，可以换成 program 吗。好像有的时候 procedure 有 function 的意思

Author: Updated in `aaf5c89`.

> - The data is not inserted manually or by other replication procedures.

Author: ```suggestion
- The data is not inserted manually or by other replication programs.
```
